### PR TITLE
C50190: Escaping @

### PR DESCRIPTION
**Pull request by v-makoud**
Hello, @dend, @mairaw,
Localization team has reported source content issue that causes localized version to have broken/different format compared to en-us version. 
Please review and merge the proposed file change to fix to target versions. If you make related fix in another PR  then share your PR number so we can confirm and close this PR.
Many thanks in advance.

## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

**Comment by mairaw** [comment]
I'm not so sure about this change @v-makoud. That section is not in Markdown format.

**Comment by Thraka** [comment]
@mairaw The english version seems to respect that markdown include that is in there. Looking at the german version, you can see that it's broken. Maybe something with the translation system does't parse markdown like it should?

I don't see why escaping @ would do anything.

**Comment by mairaw** [comment]
After merging, we need to make sure this wouldn’t be included in the IntelliSense file. That’s all. But I find odd to use something that is supposed to work in Markdown, in the ECMAXML section.

**Comment by Thraka** [comment]
Agreed, and its just the definition word, which we aren't using anymore, right? I think the proper fix is to just remove the include statement and replace it with the product name,.

**Comment by mairaw** [comment]
Ah good catch @Thraka, the include is what's broken on the localized version not the non-escaped @.

File: xml/System.Data.Objects/ObjectParameterCollection.xml [hunk 1]
```diff
@@ -200,7 +200,7 @@
         <Parameter Name="name" Type="System.String" />
       </Parameters>
       <Docs>
-        <param name="name">The name of the parameter to look for in the collection. This name should not include the "@" parameter marker that is used in the [!INCLUDE[esql](~/includes/esql-md.md)] statements, only the actual name.</param>
+        <param name="name">The name of the parameter to look for in the collection. This name should not include the "\@" parameter marker that is used in the [!INCLUDE[esql](~/includes/esql-md.md)] statements, only the actual name.</param>
```
Comment (mairaw) [inline xml/System.Data.Objects/ObjectParameterCollection.xml]: undo the change to escape the @ here and instead replace the include entry with Entity SQL

File: xml/System.Data.Objects/ObjectParameterCollection.xml [hunk 2]
```diff
@@ -310,7 +310,7 @@
         <Parameter Name="name" Type="System.String" />
       </Parameters>
       <Docs>
-        <param name="name">The name of the parameter to find. This name should not include the "@" parameter marker that is used in the [!INCLUDE[esql](~/includes/esql-md.md)] statements, only the actual name.</param>
+        <param name="name">The name of the parameter to find. This name should not include the "\@" parameter marker that is used in the [!INCLUDE[esql](~/includes/esql-md.md)] statements, only the actual name.</param>
```
Comment (mairaw) [inline xml/System.Data.Objects/ObjectParameterCollection.xml]: same here
